geo-location.html

```diff
@@ -134,6 +149,19 @@
      *   @param {Number} detail.longitude Longitude of the current position.
      */
 
+    attached: function() {
+      if ('permissions' in navigator) {
+        navigator.permissions.query({
+          name: 'geolocation'
+        }).then(function(permissionStatus) {
+          this._setPermissionStatus(permissionStatus.state);
```

Comment:
need to handle the permissions api not being available.

Reply:
@ebidel I've removed the `permission` keyword. But IMO we should keep the `api` keyword. `<geo-location>` web component is a part of my [progressive-elements](https://www.webcomponents.org/collection/FluorescentHallucinogen/progressive-elements) collection that is a set of web components for [modern web APIs](https://whatwebcando.today) such as Payment Request API, Media Session API, etc. People can find these web components by the `api` keyword.

Reply:
@ebidel What should we do if Permissions API is not available?

Reply:
Probably fail silently, but the `permissionStatus` needs a default value or some way to signal to users that permission state is unknown.